src/Extension/PhiremockProcess.php

```diff
@@ -47,6 +47,7 @@ class PhiremockProcess
      */
     public function start($ip, $port, $path, $logsPath, $debug)
     {
+        $path = realpath($path);
```

Comment:
What do you think about adding the call to realpath when the start method is being called in Phiremock.php file? That way, this method is agnostic about how the path is built.
Would also be great to call it for logsPath variable.